Argo/Types/StandardTypes.swift

```diff
@@ -81,6 +85,10 @@ public extension DictionaryLiteralConvertible where Value: Decodable, Value == V
   }
 }
 
+public func decodeObject<T: Decodable where T.DecodedType == T>(j: JSON) -> Decoded<[String: T]> {
```

Comment:
Variable Name Min Length Rule Violation: Variable name should be 2 characters or more: currently 1 characters (variable_name_min_length)